src/main/java/server/application/router/CobSpecRouter.java

```diff
@@ -1,41 +1,38 @@
 package server.application.router;
```

Comment:
Does this router still have anything to do with CobSpec?  Should it's name change?